wordpress-mobile/WordPress-Android
# Fix: updated stories lib commit hash with inexistent resource ref fix

This PR updates the stories library hash to fix in WPAndroid the issue described in https://github.com/Automattic/stories-android/issues/515

To test:
- run `./gradlew build`, observe no missing resource error appears.

PR submission checklist:
